core/src/main/scripts/importer/validateData.py

```diff
@@ -4243,6 +4243,7 @@ def processCaseListDirectory(caseListDir, cancerStudyId, logger,
                                 'all_cases_with_mutation_and_cna_data',
```

Comment:
Shouldn't `all_cases_with_fusion_data` need to be added here as well since [it is in SampleListCategory](https://github.com/cBioPortal/cbioportal/pull/6731/files#diff-d8591fa1bafcfbe2f3dc97c663e87c28R45)?

Reply:
we will replace
```
ALL_CASES_WITH_FUSION_DATA("all_cases_with_fusion_data")
```
with
```
ALL_CASES_WITH_STRUCTURAL_VARIANT_DATA("all_cases_with_structural_variant_data"),
```

Reply:
all_cases_with_fusion_data removed.

Reply:
to clarify: we are continue to use existing **_all_cases_with_fusion_data_** for fusions and **_all_cases_with_sv_data_** (newly added by PR #6860) for structural variants

Reply:
If that's the case, I don't think the `all_cases_with_sv_data ` should be deleted?

Reply:
@zhx828 , yes, we keep all_cases_with_sv_data for structural variants. It was added by PR #6860.